### ISSUE TITLE
Small typo in twisted.internet.abstract

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5101]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#5101|
|Type|defect|
|Created|2011-05-10 16:47:56Z|
|Branch|https://github.com/twisted/twisted/tree/doread-spelling-5101|

http://twistedmatrix.com/trac/browser/tags/releases/twisted-11.0.0/twisted/internet/abstract.py#L178

It's "available", not "avaliable".

I just stumbled over it. Not that I wouldn't have a life or something better to do. I just happened to have seen it...you must believe me! ;)

<details><summary>Searchable metadata</summary>

```
trac-id__5101 5101
type__defect defect
reporter__hynek hynek
priority__lowest lowest
milestone__ 
branch__branches_doread_spelling_5101 branches/doread-spelling-5101
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1305046076000000 1305046076000000
changetime__1305072116000000 1305072116000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [31796]) Branching to 'doread-spelling-5101'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [31797]) fix the typo, fix docstring formatting, news fragment

refs [#5101](https://github.com/twisted/twisted/issues/5101)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [31801]) Merge doread-spelling-5101 

Author: exarkun
Reviewer: thijs
Fixes: [#5101](https://github.com/twisted/twisted/issues/5101)

Fix a typo in the `FileDescriptor.doRead` docstring.
